src/nav2_msgs/msg/Costmap.msg

```diff
@@ -0,0 +1,9 @@
+# This represents a 2-D grid map, in which each cell has an associated cost
+
+std_msgs/Header header
+
+# MetaData for the map
+CostmapMetaData info
```

Comment:
Perhaps the field should be called "metadata" instead of "info."

Reply:
This is related to a previous comment about ROS message definitions and carryover from ROS1. Added to the discussion list.

Reply:
Fixed